mulighetsrommet-api/src/main/resources/web/openapi.yaml

```diff
@@ -935,6 +935,28 @@ paths:
                 items:
                   $ref: "#/components/schemas/SanityTiltaksgjennomforing"
 
+  /api/v1/internal/sanity/tiltaksgjennomforing/preview/{id}:
+    get:
+      tags:
+        - Sanity
+      operationId: getSanityTiltaksgjennomforingForPreview
+      parameters:
+        - in: path
+          name: id
+          required: true
+          schema:
+            type: string
+          description: ID for tiltaksgjennomføring i Sanity
+      responses:
+        200:
+          description: Hent tiltaksgjennomføring for id uavhengig av fnr i kontekst
+          content:
+            application/json:
+              schema:
+                type: array # Sanity kan returnere prod- og draft-versjon av dokument. Derfor array og ikke enkelt objekt
```

Comment:
Er dette fortsatt tilfellet etter at `Perspective` ble introdusert? https://github.com/navikt/mulighetsrommet/blob/main/mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/clients/sanity/SanityClient.kt#L87

Reply:
Ja, vi bruker `perspective = raw` for uthenting av gjennomføring, som beholder samme funksjonalitet som før perspectives ble introdusert av Sanity